sources/models/shared-defaults/lockdown-confidentiality.toml

```diff
@@ -0,0 +1,3 @@
+# Kernel
+[settings.kernel]
+lockdown = "confidentiality"
```

Comment:
I don't think we want this one - "confidentiality" isn't a setting I'd recommend.